CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 Releases
 ========
 
-v1.1.0-dev (unreleased)
+v1.0.1-dev (unreleased)
 -----------------------
 
--   No changes yet.
+-   Bugfix: Thrift: Code generated for empty services would fail to compile.
```

Comment:
In a patch release, I would expect everything to be a bug fix. Maybe the prefix isn't necessary.